### PR TITLE
Upgrade to react 16

### LINKED ISSUE #56: React 16 Support
Is there a timeline to add React 16 support?

### PR DESCRIPTION
**Pull request by reid47**
We've made changes to the package.json to allow react-tether to work with React v16.0.0

this should fix #56 

PS: we aren't sure if the yarn.lock is necessary for the upgrade.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,8 +34,8 @@
     "tether": "^1.3.7"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0"
+    "react": "^16.0.0",
```
Comment (rafeememon) [inline package.json]: Support for 0.14 and 15.x probably shouldn't be removed

Comment (reid47) [reply package.json]: @rafeememon ok will fix this ASAP.

**Comment by seanmheff** [comment]
LGTM!

**Comment by gmathieu** [comment]
Is there anything blocking this PR?

**Comment by sjolicoeur** [comment]
@rafeememon we made the necessary change, is it possible to have the PR merged and a new release?

**Comment by seanmheff** [comment]
Any update on this?

**Comment by Baldrs** [comment]
It's not going to happen until #58 is fixed

**Comment by augbog** [comment]
@souporserious are we really blocked on merging this because there isn't a new maintainer assigned yet?

**Pull request merged**